### ISSUE TITLE
[preview] docking preview editors in sidepanels does not work

### ISSUE THREAD

**Issue by vince-fugnitto** [theia-ide/theia#4719]
**Description**
Attempting to dock `preview` editors in the side-panels does not work.
If you attempt to move a `preview` editor you will notice an unrecognised file-icon and there will
be no content in the editor. If you attempt to move the editor again in any other panel, the editor is lost and the only way to recover is to reload the application.

**Reproduction Steps**
1. open an editor as a `preview` (single click).
2. move the editor to either sidepanel (left or right side panels).
3. notice the broken file-icon displayed, and no editor content.
4. move the editor to the main panel,  notice no icon, label or editor content.
5. close the editor by right clicking on the tab and closing.
6. attempt to re-open the same file from the navigator, notice that it cannot be opened anymore.

<div align='center'>

![Peek 2019-03-26 07-45](https://user-images.githubusercontent.com/40359487/54994644-1c9bab80-4f9b-11e9-8baa-062cd7247045.gif)

</div>

**Comment by akosyakov** [comment]
@vince-fugnitto Could you have a look into as well? this restyling caused so much issues 🙈 We just need to use an underlying editor icons.

**Comment by vince-fugnitto** [comment]
@akosyakov sure I can try, not sure I'll be able to tackle everything before Friday however :disappointed: 
I'm also not sure if it was introduced by the UI refresh or in master for a while, I'll take a look.

**Comment by vince-fugnitto** [comment]
@akosyakov what do you mean by the underlying editor icon? do you mean we have a single `editor` icon for all docked editors? I was also thinking we might wanna normalize the icons to the same color like the widget icons when docked.

**Comment by vince-fugnitto** [comment]
the issue has been in master for a while it seems :( docking preview-editors in the sidepanels does not work.

**Comment by akosyakov** [comment]
I don't think it is super popular use case, no one will drag editors in side panels, so the easiest fix would work. The same about normalizing colors, i doubt users would see it.

**Comment by akosyakov** [comment]
> the issue has been in master for a while it seems :( docking preview-editors in the sidepanels does not work.

i see, then no hurry, editor preview is quite buggy right now, i avoid including it in docker images and so on

**Comment by vince-fugnitto** [comment]
sounds good, I'll leave the issue for now and tackle it if ever I find some spare time :)
